src/main/java/pl/eukon05/eventboard/event/application/service/InviteToEventUseCase.java

```diff
@@ -0,0 +1,43 @@
+package pl.eukon05.eventboard.event.application.service;
+
+import lombok.RequiredArgsConstructor;
+import pl.eukon05.eventboard.common.UseCase;
+import pl.eukon05.eventboard.event.application.port.in.GetUserFriendlistPort;
+import pl.eukon05.eventboard.event.application.port.out.GetEventPort;
+import pl.eukon05.eventboard.event.application.port.out.SaveEventPort;
+import pl.eukon05.eventboard.event.domain.Event;
+import pl.eukon05.eventboard.event.domain.EventType;
+
+import java.util.List;
+import java.util.Optional;
+
+@UseCase
+@RequiredArgsConstructor
+class InviteToEventUseCase {
+    private final GetUserFriendlistPort getUserFriendlistPort;
+    private final GetEventPort getEventPort;
+    private final SaveEventPort saveEventPort;
+
+    boolean execute(String selfID, String friendID, long eventID) {
```

Comment:
Consider packing ids into value classes. (example article: https://betterprogramming.pub/why-you-should-avoid-using-primitive-types-cb55857baa39)